main.py

```diff
@@ -1,8 +1,12 @@
-from flask import Flask, render_template, request, redirect
+from flask import Flask, render_template, request, redirect, session
 from flask_mysqldb import MySQL
+import MySQLdb.cursors
  
 app = Flask(__name__)
 
+
+app.secret_key = "secret"
+
```

Comment:
What is the secret_key here? Where are you using it?

Reply:
Yes! There is no use in this statement in the application.

Reply:
Yeah! There is no use of **app.secret_key = "secret"** in the application. So I definitely remove this statement

Reply:
Without using this statement we get an error like this, **RuntimeError: The session is unavailable because no secret key was set.  Set the secret_key on the application to something unique and secret.**